test/errors/asWithUseExcept-detailed.good

```diff
@@ -0,0 +1,5 @@
+─── error in asWithUseExcept.chpl:6 [AsWithUseExcept] ───
+  Invalid use of 'as' in this use statement.
+  Cannot use 'as' with 'except'.
+  Renaming in 'except'-lists is not supported.
```

Comment:
The last two lines look pretty much the same to me. I think switching line 3 to be a code print-out (as I mentioned earlier) might make this message look nicer.